packages/public/server/__tests-e2e__/login.ts

```diff
@@ -19,49 +19,122 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
  */
-import { pages, viewports, users, elements } from './_config'
+import { pages, viewports, elements, testingServerUrl } from './_config'
 import {
   clickForNewPage,
   getByPlaceholderText,
   getBySelector,
   getByText,
+  getByRole,
   goto,
 } from './_utils'
+import { ElementHandle } from 'puppeteer'
 
-const examplePages = ['/', '/math', '/math/geometry/triangles']
+test('correct password and correct user', async () => {
+  await page.setViewport(viewports.desktop)
+  const firstPage = await goto('/math')
 
-describe('login process', () => {
-  describe.each(users)('user is %p', (user) => {
-    test.each(examplePages)('start page is %p', async (startPath) => {
-      await page.setViewport(viewports.desktop)
-      const firstPage = await goto(startPath)
+  const loginPage = await elements
+    .getLoginButton(firstPage)
+    .then(clickForNewPage)
 
-      expect(firstPage).toHaveUrlPath(startPath)
+  expect(loginPage).toHaveUrlPath(pages.login.path)
 
-      const loginPage = await elements
-        .getLoginButton(firstPage)
-        .then(clickForNewPage)
+  const { buttonLogin, inputUser, inputPassword } = pages.login.identifier
+  await typeIntoLoginForm('123456', loginPage, 'login')
```

Comment:
A structure `typeIntoLoginForm({ user: "...", password: "...", page: loginPage})` would be better to understand (you immediately understand the definition of the arguments / i.e is `'123456'` the password or the user)?. Definition:

```typescript
function typeIntoLoginForm({ page, user, password }: {page: ElementHandle, user: string, password: string }) {
...
}
```